Apps/frontend/src/views/Oscilloscope.svelte

```diff
@@ -135,6 +135,8 @@
           {#each { length: NUM_CHANNELS } as _, index}
             <TimeSweepSlider channel={index} />
           {/each}
+          <small>Common</small>
```

Comment:
Would prefer the common slider to be on top of the others. Also, can we change the color or something like that instead of/in addition to using the labels?

Reply:
+1